app/jenkins-for-jira-ui/src/components/ConnectionPanel/SetUpGuide.tsx

```diff
@@ -118,6 +132,10 @@ const SetUpGuide = (): JSX.Element => {
 		setIsDrawerOpen(false);
 	};
 
+	if (!pluginConfig) {
+		return <div>Yet to be decided...</div>;
```

Comment:
We discovered today that we only get the pluginConfig data for customers who click save in their Jenkins configuration from today on as the Save action is what currently triggers the pluginConfig event in J4J. This means that we won't have this data unless this action is taken.